Drivers/Src/BatteryBalancing.c

```diff
@@ -0,0 +1,78 @@
+
+#include "LTC6811.h"
+#include "Voltage.h"
+#include "BatteryBalancing.h"
+#include "config.h"
+#include "Current.h"
+
+#define CHARGING_TOLERANCE 0
+
+static uint16_t Balancing_Init(void) {
+	uint16_t minVoltage = 0;
+	if (Current_CheckStatus(0) == 1){
```

Comment:
`Current_CheckStatus()` returns if the current is safe or not. Use `Current_IsCharging()` to check if it is charging.